meeting/meeting-people.txt

```diff
@@ -0,0 +1,27 @@
+acozine
+andersson007_
+baptistemm
+bcoca
+briantist
+cidrblock
+cyberpear
+cybette
+dericcrago
+dmsimard
+felixfontein
+gotmax
+geerlingguy
+gundalow
+gwmngilfen
+ikhan_
+jillr
+jtanner
+lmodemal
+maxamillion
+misc
+nitzmahone
+resmo
+samccann
+tadeboro
```

Comment:
```suggestion
```

Reply:
Hmm, I thought I removed tadeboro already, but I guess not.